core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts

```diff
@@ -260,26 +264,20 @@ export class DotPageStateService {
                             siteId: page.site?.identifier
                         });
 
-                        return this.dotFavoritePageService
-                            .get({ limit: 10, userId: user.userId, url: urlParam })
-                            .pipe(
-                                take(1),
-                                catchError((error: HttpErrorResponse) => {
-                                    // Set message to throw a custom Favorite Page error message
-                                    error.error.message = this.dotMessageService.get(
-                                        'favoritePage.error.fetching.data'
-                                    );
-
-                                    this.dotHttpErrorManagerService.handle(error, true);
-
-                                    return this.setLocalPageState(page);
-                                }),
-                                switchMap((response: ESContent) => {
-                                    const favoritePage = response.jsonObjectView?.contentlets[0];
-
-                                    return this.setLocalPageState(page, favoritePage);
-                                })
-                            );
+                        return forkJoin([
+                            this.getFavoritePage(user, urlParam),
+                            this.getRunningExperiment(page.page.identifier)
```

Comment:
What happen if any of this fails?

Reply:
they will show a inline notification and in this forkjoin will return null.